### PR TITLE
fix: swap & freq_used_msg

### PR DESCRIPTION
**Pull request by JoowonYun**
- separation of swap between native and token
- add increase allowance only when provide

### CODE REVIEW

File: content/docs/howto/freq_used_msg.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Contribute to pool by sending sender's token pair. Not only increasing its size,
 
 Execute this message by the **Pair contract** address!
 
+If you provide cw20, first increase your allowance. [Execute `IncreaseAllowance`]({{< relref "/docs/reference/token" >}})
```
Comment (psy2848048) [inline content/docs/howto/freq_used_msg.md:20]: ({{< relref "/docs/reference/token#increasedecrease-allowance" >}})

Comment (JoowonYun) [reply content/docs/howto/freq_used_msg.md]: https://github.com/DELIGHT-LABS/docs-terraswap-io/pull/28/commits/ded729d15990ed69371a7287c4dfb48fdb552629

File: content/docs/howto/swap.md [hunk 2]
```diff
@@ -13,11 +13,10 @@ Therefore, a different pricing approach is used in Terraswap - algorithmic prici
 
 - Token & pair should be deployed
 - You should know the addresses of token & pair.
-- Increase your allowance. [Execute `IncreaseAllowance`]({{< relref "/docs/reference/token" >}})
 
 ## Swap by using CLI
 
-### (Native token, Contract-minted token) -> Contract-minted Token
+### When Offer_asset is native token
```
Comment (psy2848048) [inline content/docs/howto/swap.md:19]: `native token & IBC tokens` 도 이제 있어야겠죠?

Comment (JoowonYun) [reply content/docs/howto/swap.md]: https://github.com/DELIGHT-LABS/docs-terraswap-io/pull/28/commits/03dfb73a762b73988b872cf36eb0d709c83b2fc3

**Review by psy2848048 — APPROVED** [review]

**Pull request merged**